src/exporter/exporter.go

```diff
@@ -119,22 +118,59 @@ func (e *Exporter) createJobObject() error {
 		uint32(unsafe.Sizeof(jobInfo)),
 	)
 	if err != nil {
-		return fmt.Errorf("failed to set job object info:%v", err)
+		return fmt.Errorf("failed to set job object info: %v", err)
 	}
 
-	err = windows.AssignProcessToJobObject(e.jobObject, (*process)(unsafe.Pointer(e.cmd.Process)).handle)
+	handle, err := e.getHandle()
+	if err != nil {
+		return fmt.Errorf("failed to get proc handle: %v", err)
+	}
+
+	err = windows.AssignProcessToJobObject(e.jobObject, handle)
 	if err != nil {
 		return fmt.Errorf("failed to assign process to job object:%v", err)
 	}
 
 	return nil
 }
 
+func (e *Exporter) getHandle() (windows.Handle, error) {
```

Comment:
Java mannerisms are not allowed past this point :P

```suggestion
func (e *Exporter) handle() (windows.Handle, error) {
```